docs/docs/getting-started/integrating_customisations.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 3
+---
+
+# Installation
+
+The easiest way to bootstrap AEM Guides framework installation is through cli
+```bash
+npx @adobe/create-guides-extension
+```
+
+> **_NOTE:_**  Requires git & npm 
+
+# Adding customisation code
+
+1. Add code files for each component you want to extend in the `src/` directory. Some sample files have already been added for you.
+2. Now in the `index.ts` file located in the `src/` directory :
+- Add the corresponding imports for the customisations you want to add in your build.
+- Add the imports to `window.extension`
+- Register the customised component's `id` and corresponding import to `tcx extensions`
+
+# Building the customised code
+
+- Run `npm run build` in the root directory. You will get 3 files in the directory, `dist/`:
+    - `build.css`
+    - `guides-extension.js`
+    - `guides-extension.umd.cjs`
+
+- Go to `CRXDE`
```

Comment:
add crxde sample path here

Reply:
cloud users dont have access to crxde, we need to mention solution for cloud here as well

Reply:
Could you help me with the cloud solution please?

Reply:
better yet refernce link on how to do this for Customer's AEM instance (cloud or on prem)